src/js/components/List/propTypes.js

```diff
@@ -57,6 +57,7 @@ if (process.env.NODE_ENV !== 'production') {
       PropTypes.oneOfType([PropTypes.string, PropTypes.shape({})]),
     ),
     children: PropTypes.func,
+    defaultItemProps: PropTypes.shape({}),
```

Comment:
In the `index.d.ts` file we are supporting BoxTypes. I would think we'd want our propType and TypeScript definitions to have parity (i.e. be the same). Should this be `BoxPropTypes`?

Reply:
Let me test out make sure!